### DISCUSSION TITLE
Add additional permissions for commands such as /cmi unban, /cmi unjail and /cmi unmute

### DISCUSSION THREAD

**Discussion by MrCrazys** [Zrips/CMI#7482] [Enhancement]
CMI has some great punishment commands, and I am totally happy with them. However, something I would really like to see is to add a new permission node to only allow a player to revoke punishments from people who they punished. For example, the permission node for /cmi unban could be cmi.command.unban.own, which would only allow you to unban people who you have banned.

**Comment by mrfloris** [comment]
To avoid confusion in other perm nodes, where .own or .others might be used, I suggest to consider using .set or alike so ppl don't think 'oh I can unban my own ban'. 

I like it, that players can only unban those who they've banned, I would use that on my servers(s) for moderators, but allow admins to unban everybody. 

Nice suggestion.

**Reply by MrCrazys** [reply]
> To avoid confusion in other perm nodes, where .own or .others might be used, I suggest to consider using .set or alike so ppl don't think 'oh I can unban my own ban'.
> 
> I like it, that players can only unban those who they've banned, I would use that on my servers(s) for moderators, but allow admins to unban everybody.
> 
> Nice suggestion.

Yeah. Just something like I said above. Doesn't have to be the exact same, but something similar could be nice.
